osu.Game/Online/Multiplayer/MultiplayerRoomSettings.cs

```diff
@@ -39,6 +39,9 @@ public class MultiplayerRoomSettings : IEquatable<MultiplayerRoomSettings>
         [Key(7)]
         public string Password { get; set; } = string.Empty;
 
+        [Key(8)]
+        public MatchRulesetType MatchRulesetType { get; set; }
```

Comment:
As a passing thought: this may need to become something more at some point if further configurability is to be desired (i.e. more teams, customisable team names, etc.) - as I understand it right now the server is in its entirety authoritative as to how the "multi ruleset" is enforced. I presume that is out of scope for now at least (if not at all).

Reply:
Configuration is intended to be done via the event flow, not this enum.